README.md

```diff
@@ -543,10 +543,39 @@ join_field("<sourceField>", "<separator>")
 
 Looks up matching values in a map and replaces the field value with this match. External files as well as internal maps can be used.
 
+Parameters:
+
+- `path` (required): Field path to look up.
+- `map` (optional): Name or path of the map in which to look up values.
+
+Options:
+
+- `__default`: Default value to use for unknown values. (Default: Old value)
```

Comment:
is this a double `_` ?

Reply:
Yes; see [Maps.DEFAULT_MAP_KEY](https://javadoc.io/doc/org.metafacture/metamorph-api/5.4.0/constant-values.html#org.metafacture.metamorph.api.Maps.DEFAULT_MAP_KEY).